packages/eclwatch/src/WUResultStore.ts

```diff
@@ -62,7 +62,9 @@ export class Store {
                 column.formatter = (cell, row) => {
                     switch (typeof cell) {
                         case "string":
-                            return cell.replace("\t", "&nbsp;&nbsp;&nbsp;&nbsp;");
+                            return {
+                                html: cell.replace("\t", "&nbsp;&nbsp;&nbsp;&nbsp;").trim()
+                            };
```

Comment:
Not directly related to the change but this will only replace the first tab character. Maybe this would be more appropriate?
```
cell.split("\t").join("&nbsp;&nbsp;&nbsp;&nbsp;").trim()
```

Reply:
Good Point!

Reply:
See:  https://github.com/hpcc-systems/Visualization/pull/3702